src/Deserializers/LegacySnakDeserializer.php

```diff
@@ -41,11 +42,7 @@ public function deserialize( $serialization ) {
 				return new PropertySomeValueSnak( $serialization[1] );
 			case 'value':
 				return $this->deserializeValueSnak( $serialization );
-				// @codeCoverageIgnoreStart
-			default:
-				throw new LogicException();
```

Comment:
Note that this cannot happen. It is impossible that the code reaches this line of code. We should not harvest dead code.